frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -488,7 +506,7 @@ function useWMGQueryRequestBody() {
   const { data } = usePrimaryFilterDimensions();
 
   const { datasets, developmentStages, diseases, ethnicities, sexes } =
-    selectedFilters;
+    includeAllFilterOptions ? EMPTY_FILTERS : selectedFilters;
```

Comment:
Can you comment explaining this ternary? Seems like it's the crux of these changes, but I don't seem to understand the implications

Reply:
Good call out thank you! Done!